docs/DEVELOPING.md

```diff
@@ -441,6 +441,97 @@ $ make ftest NAME=sharepoint
 
 ℹ️ The connector uses the Elastic [ingest attachment processor](https://www.elastic.co/guide/en/enterprise-search/current/ingest-pipelines.html) plugin for extracting file contents. The ingest attachment processor extracts files by using the Apache text extraction library Tika. Supported file types eligible for extraction can be found as `TIKA_SUPPORTED_FILETYPES` in [utils.py](../connectors/utils.py) file.
 
+## ServiceNow Connector
+
+The [Elastic ServiceNow connector](../connectors/sources/servicenow.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+This connector is available as a **connector client** from the **Python connectors framework**. To use this connector, satisfy all [connector client requirements](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+This connector is in **beta** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+### Usage
+
+To use this connector as a **connector client**, use the **Customized connector** workflow.
+
+For additional operations, see [Usage](https://www.elastic.co/guide/en/enterprise-search/master/connectors-usage.html).
+
+### Compatibility
+
+ServiceNow "Tokyo", "San Diego" & "Rome" versions are compatible with Elastic connector frameworks.
+
+### Configuration
+
+The following configuration fields need to be provided for setting up the connector:
+
+##### `url`
+
+The host url of the ServiceNow instance.
+
+##### `username`
+
+The username of the account for ServiceNow.
+
+##### `password`
+
+The password of the account used for ServiceNow.
+
+##### `services`
+
+Comma-separated list of services to fetch data from ServiceNow. If the value is `*`, the connector will fetch data from the list of basic services provided by ServiceNow: `Users`, `Incidents`, `Request Items`, `Knowledge Articles`, `Change Requests`. Default value is `*`. Examples:
+
+  - `Incident, User`
```

Comment:
The description above mentions `Incidents`, the example `Incident`. Which one is correct?

Reply:
Updated basic services list as per ServiceNow's terminology 👍

Reply:
Please add the entire list in the first example to also show if a separator is expected. e.g. `knowledge-article` or `knowledge_article`

Reply:
Sure, I've added the entire list. Also user will have to write the entire service name as present in ServiceNow irrespective of separator.